### DISCUSSION TITLE
Shopify Collections: how to match the sorting on Shopify's end?

### DISCUSSION THREAD

**Discussion by austriker27** [statamic-rad-pack/shopify#114] [Help]
I've got a shopify collection set up for "best sellers" basically that products filtered with inventory greater than 0 and sorted by "best selling":
![image](https://user-images.githubusercontent.com/14855925/153049935-c1806683-a38d-4413-bdbd-86707a557c6f.png)

How can I retain the sorting from Shopify's end? When I output the entries on a page, it seems to display the products in alphabetical order. I don't think we have access to sales per product ([per this thread](https://github.com/jackabox/statamic-shopify/discussions/6)) so it would be nice if the output was sorted the same as it is in shopify. 

Thanks!

**Comment by ryanmitchell** [comment]
Keep an eye on https://github.com/statamic-rad-pack/shopify/pull/191
